src/techjoomla/tjnotifications/tjnotifications.php

```diff
@@ -164,14 +164,18 @@ public static function getRecipients($client,$key,$recipients,$options)
 	 *
 	 * @param   string  $body_template  A template body for email.
```

Comment:
this is no more a string now
https://scrutinizer-ci.com/g/techjoomla/TJ-Notifications/inspections/98be95b1-83c7-4d8b-92c3-c30a446ee0a8/issues/files/src/techjoomla/tjnotifications/tjnotifications.php?status=new&orderField=path&order=asc&honorSelectedPaths=0

Reply:
@manojLondhe - done